pkg/cmd/stack/translate.go

```diff
@@ -192,7 +192,7 @@ func addVolumeMountsToBuiltImage(ctx context.Context, s *model.Stack, buildKitHo
 			}
 			svc.Build = svcBuild
 			if isOktetoCluster && !strings.HasPrefix(svc.Image, "okteto.dev") {
-				tag := strings.Replace(svc.Image, ":", "-", 1)
+				tag := getNewTagFromImage(svc.Image)
```

Comment:
could we  create stackname + servicename as the image name?

Reply:
Yes, I think it's a better approach 👍